src/chrome/internal/completions/completionsRequestHandler.ts

```diff
@@ -0,0 +1,24 @@
+import { ChromeDebugLogic } from '../../chromeDebugAdapter';
+import { ICommandHandlerDeclaration, CommandHandlerDeclaration, ICommandHandlerDeclarer } from '../features/components';
+import { injectable, inject } from 'inversify';
+import { DebugProtocol } from 'vscode-debugprotocol';
+import { TYPES } from '../../dependencyInjection.ts/types';
+import { FrameParser } from '../../client/frameParser';
+
+@injectable()
+export class CompletionsRequestHandler implements ICommandHandlerDeclarer {
+    public constructor(
+        @inject(TYPES.ChromeDebugLogic) protected readonly _chromeDebugAdapter: ChromeDebugLogic,
+        private readonly _frameParser: FrameParser) { }
```

Comment:
Should this have an @inject tag?

Reply:
Because we are using the experimental metadata, I think that you don't need the inject for classes (only for interfaces)